[BrokenLinksH2] Update customer-key-manage.md

kturetz opened the pull request:
Corrected broken link on Line 103.

kturetz committed:
Update customer-key-manage.md

Corrected broken link on Line 103.

JohanFreelancer9 reviewed (changes requested):
Copy review - @kturetz, kindly commit the suggested change. Many thanks.

JohanFreelancer9 commented on microsoft-365/compliance/customer-key-manage.md:
````diff
@@ -100,7 +100,7 @@ To create a DEP to use with a mailbox, follow these steps:
    ```
 
-For detailed syntax and parameter information, see [New-DataEncryptionPolicy](/powershell/module/exchange/new-data-encryptionpolicy).
+For detailed syntax and parameter information, see [New-DataEncryptionPolicy](/powershell/module/exchange/new-dataencryptionpolicy?view=exchange-ps).
````
```suggestion
For detailed syntax and parameter information, see [New-DataEncryptionPolicy](/powershell/module/exchange/new-dataencryptionpolicy).
```

kturetz committed:
Broken Links: Update microsoft-365/compliance/customer-key-manage.md

JohanFreelancer appears to have updated my edit by forwarding the link to the correct address.

Co-authored-by: JohanFreelancer9 <48568725+JohanFreelancer9@users.noreply.github.com>

kturetz commented:
**Global effort to fix broken links** 

@krowley 

 The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. Please review within five business days and merge, or comment in the PR with any changes you'd like to see. Thanks!

JohanFreelancer9 commented on microsoft-365/compliance/customer-key-manage.md:
````diff
@@ -100,7 +100,7 @@ To create a DEP to use with a mailbox, follow these steps:
    ```
 
-For detailed syntax and parameter information, see [New-DataEncryptionPolicy](/powershell/module/exchange/new-data-encryptionpolicy).
+For detailed syntax and parameter information, see [New-DataEncryptionPolicy](/powershell/module/exchange/new-dataencryptionpolicy).
````
```suggestion
For detailed syntax and parameter information, see [New-DataEncryptionPolicy](/powershell/module/exchange/new-dataencryptionpolicy).
```

yogkumgit commented:
@tiburd Please merge this PR. Thanks

tiburd committed:
Merge branch 'public' into patch-1
